analysis/preregistration_template.Rmd

```diff
@@ -106,7 +108,7 @@ A static document for this type of process doesn't really make sense then. I pro
 
 ### 2.4.2 Data Processing and Preparation	
 
-*Describe any data preparation and processing steps, including manipulation of environmental layers, e.g. standardisatino and geographic projection.*
+*Describe any data preparation and processing steps, including manipulation of environmental layers, e.g. standardisatino and geographic projection.* Something about always retaining the raw data in a safe place, no manipulations to the raw file.
```

Comment:
Good idea. I'll add some specific items / questions about this, and provide some links in the help-text for further reading on best-practices about keeping the raw and manipulated data separate from each other, etc.

Reply:
Refs to follow up on:

- [ ] Whitlock (2011) Data archiving in ecology and evolution: best practices
- [ ] Renaut et al (2011) Management, Archiving, and Sharing for Biologists  ...
- [ ] Tierney and Ram's latest pre-print
- [ ] Preregistration templates - primary v secondary data

Reply:
```suggestion
*Describe any data preparation and processing steps, including manipulation of environmental layers, e.g. standardisation and geographic projection.* Something about always retaining the raw data in a safe place, no manipulations to the raw file.
```